pages/blog/2020-summary.md

```diff
@@ -0,0 +1,929 @@
+1. AsyncAPI showed up on [Technology Radar](https://www.thoughtworks.com/radar/tools/asyncapi) that tool 1051 users directly to AsyncAPI website
+1. Our two core tools reached their first major release
+    * [JavaScript Parser](https://github.com/asyncapi/parser-js/)
+    * [Generator](https://github.com/asyncapi/generator/)
+  
+All those moments lead us to the most important one: the [partnership with Postman](/blog/asyncapi-partners-with-postman).
+
+## Slack
+
+tl;dr AsyncAPI community activity on [our Slack workspace](https://www.asyncapi.com/slack-invite/) doubled!
+
+<Figure
+  src="/img/posts/2020-summary/active-members.webp"
+  caption="Figure 1: Slack active members weekly"
+/>
+
+### Numbers getting high
+
+I don't think the below tables require any introduction.
+
+<table>
+  <tr>
+    <td align="center">
+      
+    </td>
+    <td align="center">
+      January 2019
+    </td>
+    <td align="center">
+      January 2020
+    </td>
+    <td align="center">
+      December 2020
+    </td>
+  </tr>
+  <tr>
+    <td align="center">
+      Messages posted
+    </td>
+    <td align="center">
+      2788
+    </td>
+    <td align="center">
+      25388<br/>
+      (+22600)
+    </td>
+    <td align="center">
+      76609<br/>
+      (+51221)
+    </td>
+  </tr>
+    <tr>
+    <td align="center">
+      Number of members
+    </td>
+    <td align="center">
+      159
+    </td>
+    <td align="center">
+      432<br/>
+      (+273)
+    </td>
+    <td align="center">
+      888<br/>
+      (+456)
+    </td>
+  </tr>
+</table>
+
+More members means higher activity :sweat_smile:
+
+<table>
+  <tr>
+    <td align="center">
+    </td>
+    <td align="center">
+      2019
+    </td>
+    <td align="center">
+      2020
+    </td>
+  </tr>
+  <tr>
+    <td align="center">
+      Weekly members activity 
+    </td>
+    <td align="center">
+      45 average<br/>
+      42 median
+    </td>
+    <td align="center">
+      81 average<br/>
+      76 median
+    </td>
+  </tr>
+</table>
+
+### Most present companies
+
+I was also super curious to check out where our members are coming from. The only way to do it was by checking the domain of their emails used during registration. Yes, this is not 100% accurate. Not all people use company emails in open source. It is visible in data; half of the users have the **gmail.com** domain. Nevertheless, the result of my investigation is pretty accurate with our knowledge. The ones that have more than three representants:
+- mulesoft.com
+- postman.com
+- redhat.com
+- sap.com
+- smartbear.com
+- solace.com
+- tibco.com
+- worldpay.com
+
+Some of these companies sponsored our work in 2020. The ones that didn't... don't worry, we'll contact you :wink:. In 2021 we want to go into a neutral ground (join a foundation). There'll be no more excuses.
```

Comment:
A bit offensive? 😆

Reply:
offensive for who? corporations? more than my "fuck you" for covid?

Reply:
`don't worry, we'll contact you :wink:` this part 😄

Reply:
The covid part was fine I think :D

Reply:
nah, this is why I wink 😄 I'm not gonna chase anybody with our piggy bank, I'm more curious about their use cases and their plans for AsyncAPI and how they could help us out